sotodlib/hwp/hwp.py

```diff
@@ -220,18 +285,19 @@ def fit_func(x, A0, As, Bs, N_mode, N, sins, coss):
 
     for i in range(aman.dets.count):
         params_0 = list(As[:, i]) + list(Bs[:, i]) + list(modes)
-        params_0.append(np.mean(aman[signal][i, m[i]]))
+        params_0.append(np.median(aman[signal][i, m[i]]))
```

Comment:
any reason not to make this `nanmedian` instead of `median`? (I often use `nanmedian` when I just want to blindly not have to worry about edge cases).